blog/models.py

```diff
@@ -0,0 +1,36 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class Post(models.Model):
+    owner = models.ForeignKey("User", on_delete=models.CASCADE)
+    title = models.CharField(max_length=200)
+    content = models.TextField()
+    created_time = models.DateTimeField(auto_now_add=True)
+
+    def __str__(self):
+        return self.title
+
+    @property
+    def comment_count(self):
+        return self.commentary_set.count()
```

Comment:
The related_name for the ForeignKey in the Commentary model should be specified to avoid the default related_name 'commentary_set'. This will make the code more readable and maintainable.